apps/server/src/modules/learnroom/service/common-cartridge-export.service.ts

```diff
@@ -82,21 +78,43 @@ export class CommonCartridgeExportService {
 			};
 		}
 
-		if (content.component === ComponentType.GEOGEBRA) {
+		if (version === CommonCartridgeVersion.V_1_3_0 && content.component === ComponentType.GEOGEBRA) {
```

Comment:
The following ifs looks like a code duplication

Reply:
Unfortunately, there is no other solution to avoid duplication here. Otherwise the CommonTags will be displayed in the xml file as xml tag.